README.md

```diff
@@ -1160,3 +1160,67 @@ Possible roles:
 If a user has access through multiple sources (e.g. they belong to
 two teams and also have direct project membership) they will have
 the highest privileges afforded by any of those access routes.
+
+
+
+## Metrics Importing Functionality
+
+It is possible to use Morgue to configure importers for stability score.
+Usage:
+
+```
+morgue metrics-import <command>...
+```
+
+### `source check-query`
+
+Determines if a query is valid by running it against a source as if it had been
+used with an importer and displays diagnostic information.  For example:
+
+```
+morgue metrics-importer source check-query --source my-source-uuid \
+--query 'select time, value from test where time >= $1 and time < $2'
+```
+
+
+### `importer create`
+
+Creates an importer.  Takes the following options:
+
+Option       |
```

Comment:
Does this line up correctly outside the diff view? I see one missing space after `Option`